src/api/app/views/webui/cloud/azure/configurations/show.html.haml

```diff
@@ -0,0 +1,90 @@
+%h1
+  Microsoft Azure Configuration
+
+%p
+  You need to provide an application id and an application key in order to allow OBS to upload images to your Azure account.
+  We will encrypt your configuration before it get stored in our database.
+
+%h2 How to Generate an Application ID and an Application Key
```

Comment:
Phew. Whenever some small detail in the workflow on azure changes we will have to update this. Is there not a link you can point to?

Reply:
I never have seen any link that explains this procedure. @mdeniz maybe you have one?

Reply:
Maybe we should create on then. On the openSUSE wiki or some page on openbuildservice.org

Reply:
My reasoning is this: What if we release this in OBS 2.10 and it changes? Are we going to release a maintenance update because Microsoft renames a section header in azure?

Reply:
I didn't consider this but given the fact that they just introduced AD as the new API, I don't think they change it soon.

But I'm open to point to something that is more suitable.

Reply:
I would put the steps [into our documentation](http://openbuildservice.org/help/manuals/obs-admin-guide/obs.cha.installation_and_configuration.html#_cloud_upload_setup)